source/install.txt

```diff
@@ -0,0 +1,80 @@
+.. _mdb-shell-install:
+
+=======================
+Install the |mdb-shell|
+=======================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+.. include:: /includes/admonitions/fact-mdb-shell-beta.rst
+
+Prerequisites
+-------------
+
+To use the |mdb-shell|, you must have a MongoDB deployment to connect
+to. 
+
+- For a free cloud-hosted deployment, you can use
+  `MongoDB Atlas <https://www.mongodb.com/cloud/atlas?tck=docs_vsce>`__.
+
+- To learn how to run a local MongoDB deployment, see
+  :manual:`Install MongoDB </installation/>`.
+
+Procedure
+---------
+
+.. tabs-platforms::
+
+   .. tab::
+      :tabid: windows
+
+      .. include:: /includes/steps/install-shell-windows.rst
+
+   .. tab::
+      :tabid: macOS
+
+      .. _macos-install-homebrew:
+
+      Install with Homebrew
+      ~~~~~~~~~~~~~~~~~~~~~
+
+      .. important::
+
+         To install ``mongosh`` with Homebrew on macOS, you must have a
+         minimum `node.JS <https://nodejs.org/en/download/>`__ version
+         of ``12.0.0``.
```

Comment:
- Homebrew itself also has requirements: https://docs.brew.sh/Installation
- Not sure how much you want to include. If you direct users to the Homebrew install guide that lists these requirements, I think you don't need to list them here.

Reply:
Agreed. Added a specific blurb to this which links to that page.